tools/syz-cover/syz-cover.go

```diff
@@ -67,7 +67,8 @@ func main() {
 	if err != nil {
 		tool.Fail(err)
 	}
-	rg, err := cover.MakeReportGenerator(target, *flagVM, *flagKernelObj, *flagKernelSrc, *flagKernelBuildSrc)
+	var subsystem []cover.Subsystem
+	rg, err := cover.MakeReportGenerator(target, *flagVM, *flagKernelObj, *flagKernelSrc, *flagKernelBuildSrc, subsystem)
```

Comment:
It's simpler to pass `nil` here.
However, the slice is empty and does not contain the "all" subsystem that is supposed to be present.
I think it's better to move append of "all" into pkg/cover, then tools/syz-cover will have it as well.